include/simdjson/error.h

```diff
@@ -37,6 +37,7 @@ enum error_code {
   PARSER_IN_USE,            ///< parser is already in use.
   OUT_OF_ORDER_ITERATION,   ///< tried to iterate an array or object out of order
   INSUFFICIENT_PADDING,     ///< The JSON doesn't have enough padding for simdjson to safely parse it.
+  UNALLOWED_CAPACITY,       ///< We are requesting more than SIMDJSON_MAXSIZE_BYTES bytes of capacity
```

Comment:
I think CAPACITY already covers this, doesn't it? Might be a small distinction I'm not catching, though.

Reply:
Oh. If we do need a new one, s/UNALLOWED/DISALLOWED I think is the proper English.